.github/workflows/release-branch.yml

```diff
@@ -27,6 +27,13 @@ jobs:
               if: github.event_name != 'schedule' || fromJSON(steps.check-date.outputs.result).create_release_branch
               run: python scripts/make_release_branch.py
 
+            - name: Install python packages
+              run: pip install -r scripts/requirements.txt
+
+            - name: Send slack message about branch
+              if: github.event_name != 'schedule' || fromJSON(steps.check-date.outputs.result).create_release_branch
+              run: python scripts/send_slack_message.py --token ${{ secrets.WORKFLOW_GITHUB_TOKEN }} --webhook ${{ secrets.SLACK_WEBHOOK_URL }} --slack_ids ${{ secrets.USERNAMES}}
```

Comment:
Maybe it's better to convert `send_slack_message` to `make_release_branch` as a function?
It may help to avoid:
- duplicating `if` expression
- coming up with new name for `send_slack_message` - the current one is to generic